soc/arm/raspberrypi/rp2/Kconfig.defconfig.series

```diff
@@ -0,0 +1,46 @@
+# Raspberry Pi RP20XX MCU line
+
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# Copyright (c) 2021 Yonatan Schachter
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_RP2XXX
+
+source "soc/arm/raspberrypi/rp2/Kconfig.defconfig.rp2*"
+
+config SOC_SERIES
+	default "rp2"
+
+config NUM_IRQS
+	default 26
+
+choice
+	prompt "Raspberry Pi flash type"
```

Comment:
we should not have visible symbols in a defconfig.
Why is this not in the modules Kconfig file ?

And which flash type is the correct default to chose for a board ?
I don't assume the user should select / reconfigure the flash type, correct ?